tools/compile_clang.sh

```diff
@@ -54,6 +54,22 @@ WARN_PACKAGE () {
     fi
 }
 
+DOWNLOAD_HELPER() {
+    WGET_CTR=1
+    while [ 1 ]; do
+        wget --no-dns-cache --tries=3 --retry-connrefused --waitretry=3 $1 >/dev/null 2>&1
+        if [[ $? == 0 ]]; then
+            break;
+        else
+            if [[ $WGET_CTR == 3 ]]; then
+              exit 1
+            fi
+            WGET_CTR=$($WGET_CTR + 1)
+            echo "${WGET_CTR}. try...."
```

Comment:
nit: "wget $1 failed attempt $WGET_CTR. Retrying..." ?

Reply:
currently it prints `3. try...` well, it must have failed right?

Reply:
++ we are not interested for whatever reason the download was failed. (as long as it works eventually)

Reply:
Yeah, but I would argue theres no reason to not have a more descriptive sentence here if it helps future trouble shooting or general developer experience.

Reply:
Could be. IMHO the output from WGET is more useful than anything we might say here. 

FWIW, after this PR, this tool will exit with the right exit code in case one of the packages above wasn't downloaded properly. Package creator will recognize the issue now.